README.md

```diff
@@ -1,199 +1,113 @@
+    - **Google users**: the [Terraform docs](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_reference#full-reference) explain the options, e.g. using `GOOGLE_CREDENTIALS`.
 
-1. Enable pull request build triggers: from your Azure DevOps organization, click on your project > Project Settings > Repositories > your repository > click on the Policies tab > under the Branch Policies, click on your default branch > if you don't already have a Build Validation, click on the + sign to add one. To add one: select your Pipeline, leave path filter blank, set the Trigger to Automatic, and Policy requirement to Optional (your can also use Required but we don't recommend it). Without this, Azure DevOps Pipelines do not trigger builds with the pull request ID, thus comments cannot be posted by the integration.
+5.  Add the following to the `azure-pipelines.yml` file:
 
-2. Enable DevOps Pipelines to post pull request comments: from your Azure DevOps organization, click on your project > Project Settings > Repositories, click on the Securities tab, under Users click on the "[project name] Build Service ([org name])" user, and set the "Contribute to pull requests" to Allow.
+    ```yaml
+    pr:
+      - master
 
-3. Add secret variables: from your Azure DevOps organization, click on your project > Pipelines > your pipeline > Edit > Variables, and click the + sign to add variables for the following. Also tick the "Keep this value secret" option.
-    - `SYSTEM_ACCESSTOKEN`: leave this as `$(System.AccessToken)` so the [default REST API token](https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#system-variables-devops-services) is available to post pull request comments.
-    - `INFRACOST_API_KEY`: your Infracost API key. To get an API key [download Infracost](https://www.infracost.io/docs/#quick-start) and run `infracost register`.
-    - [Azure credentials](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/guides/service_principal_client_secret) so Terraform init can be run. Usually these are `ARM_CLIENT_ID`, `ARM_CLIENT_SECRET`, `ARM_SUBSCRIPTION_ID` and `ARM_TENANT_ID`.
+    variables:
+      TF_ROOT: PATH/TO/TERRAFORM/CODE
 
-4. Add the following stage to your `azure-pipelines.yml` (create this file if you don't have one already). You can customize the `env` section to add variables and environment variables mentioned in the following sections. If you're using a self-hosted agent, please ensure that `bc`, `curl` and `git` are installed as the `diff.sh` script uses those.
+    jobs:
+      - job: terraform_plan_json
+        displayName: Terraform plan JSON
+        pool:
+          vmImage: ubuntu-latest
 
-    ```yml
-    pool:
-      vmImage: ubuntu-latest
+        steps:
+          - task: TerraformInstaller@0
```

Comment:
@aliscott just thought - we should add a comment/step for this to let the users know how to get the `TerafformInstaller` task (microsoft terraform extension)